recipe/meta.yaml

```diff
@@ -28,9 +32,11 @@ test:
         - test -f $PREFIX/lib/libflann_s.a                                        # [unix]
         - test -f $PREFIX/lib/libflann.dylib                                      # [osx]
         - test -f $PREFIX/lib/libflann.so                                         # [linux]
+        - test -f $PREFIX/lib/libflann.dll                                        # [win]
         - test -f $PREFIX/lib/libflann_cpp_s.a                                    # [unix]
         - test -f $PREFIX/lib/libflann_cpp.dylib                                  # [osx]
         - test -f $PREFIX/lib/libflann_cpp.so                                     # [linux]
+        - test -f $PREFIX/lib/libflann_cpp.dll                                    # [osx]
```

Comment:
Should add there is no `test` on Windows. Also, these will all live in `%LIBRARY_PREFIX%`. Take a look at this [example]( https://github.com/conda-forge/hdf5-feedstock/blob/master/recipe/meta.yaml#L87-L88 ) from the `hdf5` recipe.